iina/PlaybackHistory.swift

```diff
@@ -15,7 +15,13 @@ fileprivate let KeyPlayed = "IINAPHPlayed"
 fileprivate let KeyAddedDate = "IINAPHDate"
 fileprivate let KeyDuration = "IINAPHDuration"
 
-class PlaybackHistory: NSObject, NSCoding {
+/// An entry in the playback history file.
+/// - Important: This class conforms to [NSSecureCoding](https://developer.apple.com/documentation/foundation/nssecurecoding).
+///     When making changes be certain the requirements for secure coding are not violated by the changes.
+class PlaybackHistory: NSObject, NSCoding, NSSecureCoding {
```

Comment:
`NSSecureCoding` already inherits from `NSCoding`, do we need to confirm these two at the same time?

Reply:
Sorry I missed that. Fixed.